src/test/java/jenkins/branch/OrganizationFolderTest.java

```diff
@@ -65,7 +75,23 @@ public void configRoundTrip() throws Exception {
         assertEquals(MockFactory.class, projectFactories.get(0).getClass());
         assertEquals(MockFactory.class, projectFactories.get(1).getClass());
     }
-    @TestExtension("configRoundTrip")
+
+    @Test
+    public void indexChildrenOnOrganizationFolderIndex() throws Exception {
+        OrganizationFolder top = r.jenkins.createProject(OrganizationFolder.class, "top");
+        List<MultiBranchProjectFactory> projectFactories = top.getProjectFactories();
+        projectFactories.add(new MockFactory());
+        top.getNavigators().add(new SingleSCMNavigator("stuff", Collections.<SCMSource>singletonList(new SingleSCMSource("id", "stuffy", new NullSCM()))));
+        top = r.configRoundtrip(top);
+
+        RingBufferLogHandler logs = createJULTestHandler(); // switch to the mock log handler
+
+        top.setDescription("Org folder test");
+        top = r.configRoundtrip(top);
+        waitForLogFileMessage("Indexing multibranch project: stuff", logs);
```

Comment:
:ant: I think rather than waiting for log messages, you can call `getIndexing().getTimestamp()` and compare to the original.